airflow/models/taskinstance.py

```diff
@@ -851,6 +852,7 @@ def _check_and_change_state_before_execution(
         return True
 
     @provide_session
+    @Sentry.format_run_task
```

Comment:
I think I'd like to come up with a better name for this wrapper than `format_run_task` (it's the `format` part I don't like as I don't know what this _does_)

So what does this decorator do?

Is `@Sentry.record_tags` or `@Sentry.track_context` a better name?

Reply:
Or `@Sentry.enrich_errors`?

Reply:
`@Sentry.enrich_errors` sounds really good!